.github/workflows/pytest.yml

```diff
@@ -0,0 +1,79 @@
+name: PyTest
+on: push
+
+env:
+  TEST_DATA_REF: "https://github.com/valentinaschueller/ece-4-monitoring-test-data/archive/refs/heads/main.zip"
+  TEST_DATA_DOWNLOAD: "test-data"
+
+jobs:
+    pytest:
+      runs-on: ubuntu-latest
+      defaults:
+        run:
+          shell: bash -l {0}
+      strategy:
+        matrix:
+          python-version: [ '3.7', '3.8', '3.9', '3.10' ]
+
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v3
+          with:
+            ref: 62-github-actions
+            
+        - name: Setup Python
+          uses: actions/setup-python@v3
+          with:
+            python-version: ${{matrix.python-version}}
+        
+        - name: Setup miniconda
+          uses: conda-incubator/setup-miniconda@v2
+          with:
+            miniconda-version: "latest"
+            auto-activate-base: true
+            environment-file: conda_environment.yml
+            python-version: ${{matrix.python-version}}
+
+        - name: Finish conda setup
+          run: |
+            conda info
+            conda install pip
+            python -m pip install --upgrade pip
+            conda list
+
+        - name: Install package + remaining PyPI dependencies
+          run: pip install .
```

Comment:
Again, out of interest: So we do not have to use `pip install --editable` here? I know we do not need the development setting here, but I wasn't aware that pip installs from a source directory in that way.

Reply:
I think I just dropped it since we don't edit the folder, and never faced an issue. But yeah, it's possible, as mentioned e.g. [here](https://stackoverflow.com/a/41537134/11247528)

Reply:
Absolutely right, we do not need `-e`, I just wasn't aware it was possible. Good day, I learned something :smile: